doc/sphinx/source/contributing/dev_install.rst

```diff
@@ -0,0 +1,68 @@
+.. _source:
+
+Development installation on Linux or macOS
+===========================================
+
+If you intend to contribute to the NNPDF code,
+you will need to install a development version of the code.
+This is necessary if you intend to add new features, data or modify existing tools.
+
+Code development is carried out using Github.
+For more information on the Git workflow that NNPDF adopts, see the section :ref:`gitsection`.
+
+While we recommend that you install ``nnpdf`` in the most comfortable way for your workflow,
+installing some of the dependencies such as LHAPDF might be challenging.
+Below we describe how to prepare a development environment using conda.
+
+1. Setup a conda environment with a recent python version supported by NNPDF (see `here <https://github.com/NNPDF/nnpdf/blob/master/pyproject.toml>`_).
+
+.. code::
+
+   conda create -n nnpdf_dev lhapdf pandoc python==3.12 -c conda-forge
+
+
+2. Clone the repository
+
+.. code::
+
+    git clone https://github.com/NNPDF/nnpdf.git
+    cd nnpdf
+
+
+Now choose how to install the dependencies of the code.
+For a pip-only installation, go to 3., if you prefer to use conda go to 4.
```

Comment:
If you want to stop pushing conda at every commit to master, we also shouldn't do conda here

Reply:
Conda to install the dependencies is still ok.

Reply:
True, but it is a good baseline though.